src/web/app/pages-instructor/instructor-courses-page/instructor-courses-page.component.html

```diff
@@ -1,52 +1,55 @@
+                   ngbTooltip="Enter the identifier of the course, e.g.CS3215-2013Semester1."
+                   [maxlength]="40" placeholder="e.g. CS3215-2013Semester1"/>
+          </div>
         </div>
-      </div>
-      <div class="form-group row">
-        <label class="col-sm-3 control-label bold-label text-right">Course Name:</label>
-        <div class="col-sm-9">
-          <input class="form-control" type="text" [(ngModel)]="newCourseName"
-                 ngbTooltip="Enter the name of the course, e.g. Software Engineering."
-                 [maxlength]="64" placeholder="e.g. Software Engineering"/>
+        <div class="form-group row">
+          <label class="col-sm-3 control-label bold-label text-right">Course Name:</label>
+          <div class="col-sm-9">
+            <input class="form-control" type="text" [(ngModel)]="newCourseName"
+                   ngbTooltip="Enter the name of the course, e.g. Software Engineering."
+                   [maxlength]="64" placeholder="e.g. Software Engineering"/>
+          </div>
         </div>
-      </div>
-      <div class="form-group row">
-        <label class="col-sm-3 control-label bold-label text-right">Time Zone:</label>
-        <div class="col-sm-9">
-          <div class="input-group">
-            <select class="form-control" [(ngModel)]="timezone" ngbTooltip="The time zone for the course. You should not
+        <div class="form-group row">
+          <label class="col-sm-3 control-label bold-label text-right">Time Zone:</label>
+          <div class="col-sm-9">
+            <div class="input-group">
+              <select class="form-control" [(ngModel)]="timezone" ngbTooltip="The time zone for the course. You should not
                  need to change this as it is auto-detected based on your device settings. TEAMMATES automatically adjusts
                  to match the current time offset in your area, including clock changes due to daylight saving time.">
-              <option *ngFor="let timezone of timezones" [value]="timezone">{{ timezone }}</option>
-            </select>
-            <span class="input-group-btn">
+                <option *ngFor="let timezone of timezones" [value]="timezone">{{ timezone }}</option>
+              </select>
+              <span class="input-group-btn">
               <input type="button" class="btn btn-primary" value="Auto-Detect" (click)="onAutoDetectTimezone()">
             </span>
+            </div>
           </div>
         </div>
-      </div>
-      <div class="form-group row">
-        <div class="offset-sm-3 col-sm-9">
-          <input type="button" class="btn btn-primary" (click)="onSubmit()" value="Add Course">
+        <div class="form-group row">
+          <div class="offset-sm-3 col-sm-9">
+            <input type="button" class="btn btn-primary" (click)="onSubmit()" value="Add Course">
+          </div>
         </div>
       </div>
     </div>
-  </div>
-</ng-container>
+  </ng-container>
+</div>
 
```

Comment:
I don't think the white space between the 2 containers is necessary, can remove.